### PR TITLE
Prepares for application emails 

### PR DESCRIPTION
**Pull request by discorick**
The lengthy discussion surround this PR can be found at #211

### CODE REVIEW

File: vendor/engines/saas/app/jobs/base_login_job.rb [hunk 1]
```diff
@@ -22,7 +28,7 @@ def map_payload(params)
     end
 
     params['user']['emails'] = params['emails']
-    params['user']['action'] = self.instance_variable_get('@action')
+    params['user']['action'] = self.class.instance_variable_get('@action')
```
Comment (dahlbyk) [inline vendor/engines/saas/app/jobs/base_login_job.rb]: Isn't this code already working in production?

**Comment by dahlbyk** [comment]
> fixes some code lost in the merge

Sorry for the bad merge, should have just let you do it. :frowning:

**Comment by dahlbyk** [comment]
Nothing new to add here. Is there anything in particular I need to be testing, or you've tested that you can share (e.g. saving into Couch).

**Comment by discorick** [comment]
The new couch docs look like this: 
image

File: vendor/engines/saas/app/jobs/base_login_job.rb (hunk 1 above)
Comment (discorick) [inline vendor/engines/saas/app/jobs/base_login_job.rb]: It is not :(

**Comment by discorick** [comment]
No worries about the merge - was easy to sort out

File: vendor/engines/saas/app/jobs/base_login_job.rb (hunk 1 above)
Comment (dahlbyk) [inline vendor/engines/saas/app/jobs/base_login_job.rb]: Right you are - confirmed on Identify payloads in prod and dev.

**Pull request merged**